Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "mauth-client"
 version = "0.1.0"
```

Comment:
Can you bump the version as well, so I can push the updated version to crates.io?

Reply:
I thought of that but then, there are no changes really, I think not worth a version. This is mostly so it does not bitrot and when features come is in a good state

Reply:
That sounds right actually when I checked again how Rust Editions actually work 👍